backend/app/assets/stylesheets/spree/backend/globals/_variables.scss

```diff
@@ -206,3 +207,14 @@ $z-index-navbar-flyout:          1000 !default;
 // Sidebar
 //--------------------------------------------------------------
 $border-sidebar:                 1px solid $color-sidebar-border !default;
+
+// Media queries
+//--------------------------------------------------------------
+$desktop-width: 959px !default;
+$tablet-width: 768px !default;
+$mobile-width: 480px !default;
+
+$large-device: "only screen and (min-width: #{$tablet-width})";
+$desktop: "only screen and (max-width: #{$desktop-width})";
+$tablet: "only screen and (max-width: #{$tablet-width})";
+$mobile: "only screen and (max-width: #{$mobile-width})";
```

Comment:
The link are pointing to is for the frontend. 

And yes, hopefully they should match, otherwise we need them to match, otherwise we can't use Bootstrap ..

So, [this must be the bootstrap configuration](https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/_bootstrap_custom.scss#L107-L119) for the backend.

The [skeleton one for the backend](https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/shared/_skeleton.scss#L132) seems to have most of the values pretty fixed : ) .. so yeah, hopefully they match ..